ContentPatcher/docs/i18n/zh/author-guide/action-editdata.md

```diff
@@ -0,0 +1,508 @@
+ID为`{{ModId}}_Pufferchick`的物品，会被添加到`Data/Objects`中：
+
+```js
+{
+    "Format": "2.7.0",
+    "Changes": [
+        {
+            "Action": "EditData",
+            "Target": "Data/Objects",
+            "Entries": {
+                "{{ModId}}_Pufferchick": {
+                    "Name": "{{ModId}}_Pufferchick",
+                    "DisplayName": "Pufferchick",
+                    "Description": "An example object.",
+                    "Type": "Seeds",
+                    "Category": -74,
+                    "Price": 1200,
+                    "Texture": "Mods/{{ModId}}/Objects"
+                }
+            },
+        }
+    ]
+}
+```
+
+你还可以编辑条目中的字段。当条目是字符串时，该值会被假定为一个以/分隔的字段列表（每个字段都有一个从0开始的数字），否则字段是给定条目中的直接条目。
+
+比如，这将编辑一个条目的描述字段：
+
+```js
+{
+    "Format": "2.7.0",
+    "Changes": [
+        {
+            "Action": "EditData",
+            "Target": "Data/Objects",
+            "Fields": {
+                "MossSoup": { //ID为"MossSoup"的条目
+                    "Description": "Maybe a pufferchick would like this."
+                }
+            }
+        },
+    ]
+}
+```
+
+你可以用把字段设为`null`的方式把某个条目删除。比如，下列事件会被替换为一个前置条件不同的另一个事件：
+```js
+{
+    "Format": "2.7.0",
+    "Changes": [
+        {
+            "Action": "EditData",
+            "Target": "Data/Events/Beach",
+            "Entries": {
+                "733330/f Sam 750/w sunny/t 700 1500/z winter/y 1": null,
+                "733330/f Sam 750/w sunny/t 700 1500/z winter": "事件脚本"
+            }
+        }
+    ]
+}
+```
+
+当一个值有嵌套的条目时，你可以用[`目标字段`](#目标字段2)来编辑特定的那一个。
+
+### 编辑列表<a name="edit-a-list"></a>
+你可以用同样的方法编辑[列表](#data-assets)。
+
+在游戏原版的数据素材中，列表没有键，但在Content Patcher中，他们仍有一个键来实现`Entries`和`MoveEntries`。也就是说，编辑列表和编辑字典差不多。
+
+对于模型列表（`{ ... }`块），键是每个模型的`Id`字段。比如，`Data\LocationContexts`就展示了ID为`spring1`的`Music`条目：
+```js
+{
+    "Default": {
+        "SeasonOverride": null,
+        "DefaultMusic": null,
+        "DefaultMusicCondition": null,
+        "DefaultMusicDelayOneScreen": true,
+        "Music": [
+            {
+                "Id": "spring1",
+                "Track": "spring1",
+                "Condition": "SEASON Spring"
+            }
+            ...
+        ]
+    }
+}
+```
+
+为了更改那个内容包中的音乐，你必须使用它的ID：
+```js
+{
+    "Format": "2.7.0",
+    "Changes": [
+        {
+            "Action": "EditData",
+            "Target": "Data/LocationContexts",
+            "TargetField": [ "Default", "Music" ],
+            "Entries": {
+                "spring1": {
+                    "Id": "spring1",
+                    "Track": "spring1",
+                    "Condition": "SEASON Spring, YEAR 2"
+                }
+            }
+        }
+    ]
+}
+```
+
+编辑简单字符串列表的方法和上述完全相同，只是字符串本身就是键。详见[编辑对象上下文标签示例](#编辑对象上下文标签示例)。
+
+### 移动列表条目<a name="moving-list-entries"></a>
+对于列表来说，顺序很重要（比如，游戏会使用`Data\MoviesReactions`的第一个条件适合的条目来匹配NPC反应。你可以用`MoveEntries`字段来更改顺序。比如，这会移动`Abigail`的每一个条目：
```

Comment:
> **en:** For example, this moves the `Abigail` entry using each possible operation:
> **zh:** 比如，这会移动`Abigail`的每一个条目：

The translation seems to say it will move "every entry of `Abigail`", but it will only move one; it's assumed that IDs are unique.